src/super_gradients/common/aws_connection/aws_secrets_manager_connector.py

```diff
@@ -95,18 +96,26 @@ def __get_secrets_manager_dict_for_secret_name(aws_env: str, secret_name: str) -
 
         try:
             if not AWSSecretsManagerConnector.current_environment_client:
-                logger.debug('Initializing a new secrets manager client...')
+                logger.debug("Initializing a new secrets manager client...")
                 AWSSecretsManagerConnector.current_environment_client = AWSConnector.get_aws_client_for_service_name(
-                    profile_name=aws_env,
-                    service_name='secretsmanager')
+                    profile_name=aws_env, service_name="secretsmanager"
+                )
             logger.debug(f'Fetching the secret "{secret_name}" in env "{aws_env}"')
             aws_secrets = AWSSecretsManagerConnector.current_environment_client.get_secret_value(SecretId=secrets_path)
-            aws_secrets_dict = json.loads(aws_secrets['SecretString'])
+            aws_secrets_dict = json.loads(aws_secrets["SecretString"])
             return aws_secrets_dict
 
         except Exception as ex:
-            error = '[' + current_class_name + '] - Caught Exception while trying to connect to aws to get credentials from secrets manager: ' + '"' + str(
-                ex) + '"' + ' for ' + str(secrets_path)
+            error = (
```

Comment:
I know it's just some black formatter automatic thing, but can you please fix this ugly ++++ and replace it with a formatted string?

Reply:
Fixed in 20ac8a290ce023b49df1cb9d67eb77c663a792d1